bundle/language_en.properties

```diff
@@ -798,7 +994,6 @@ angal.common.description
 angal.common.descriptionm                                                        = DESCRIPTION
```

Comment:
convert to lowercase

Reply:
This will be taken care of in my next pass